rllib/examples/inference_and_serving/policy_inference_after_training_with_lstm.py

```diff
@@ -56,7 +56,7 @@
 parser.add_argument(
     "--stop-reward",
     type=float,
-    default=150.0,
+    default=0.8,
```

Comment:
FrozenLake will not reach 150 :)

Reply:
dropping from 150 to 0.8?? is this a unit test or something? why do we even bother updating these flag defaults with this PR?

Reply:
you are right, this one-line fix should not be in this PR, but I noticed it b/c this test was failing initially on this PR. Frozenlake has a max reward of 1.0 so 150 doesn't make any sense here (it's a leftover from the other CartPole-based examples).